app/views/users/edit.html.erb

```diff
@@ -0,0 +1,15 @@
+<% content_for :title, "#{t(".title")} #{@user.name}" %>
+
+<% content_for :action_bar do %>
+  <%= render "shared/link_github_account", user: @user %>
+  <%= render "shared/link_google_account", user: @user %>
+  <%= link_to t(".cancel"), user_path(@user), class: "button" %>
+<% end %>
+
+<%= form_for @user, html: { autocomplete: "off" } do |f| %>
```

Comment:
We don't put `space` inside hashes.

So, `html: { autocomplete: "off" }` should be `html: {autocomplete: "off"}`.

I will fix later if any.